[Hackday] Retrospectivas (#302)
elo7/tech-blog: content/posts/retrospectivas-e-sua-importancia.md

---
title: Retrospectivas e sua importância
date: 2020-11-30
category: front-end
layout: post
description: Neste post discutiremos os benefícios da adoção de retrospectivas no seu time.
authors: [mwingter]
tags:
  - retrospectivas
  - agile
  - metodologias
---

Uma retrospectiva é uma parte essencial da metodologia ágil, que tem como um de seus pilares a melhoria contínua na produtividade do time. O objetivo deste post é justamente mostrar a importância desta prática e como ela funciona, dando alguns exemplos de técnicas e de ferramentas online que nós do Elo7 temos utilizado para realizar a prática remotamente.

# Retrospectivas
O objetivo principal de uma retrospectiva é avaliar todo o processo de trabalho ao final de cada sprint. Em outras palavras, isto significa um verdadeiro "lavar roupa suja", ou seja, discutir o que deu ou não deu certo, o que poderia ser melhorado, o que continuar fazendo ou deixar de fazer, e com isto entender os pontos fortes e fracos e implementar estratégias e ações para superá-los e, assim, garantir que a melhoria contínua aconteça, gerando um aumento na produtividade e na qualidade e uma redução de falhas.

## Porque fazer retrospectivas?

> Uma das principais razões para se fazer retrospectivas é a nova perspectiva que os membros ganham ao fazer autoavaliações, podendo discutir e ouvir vários pontos de vistas diferentes sobre si e o projeto em questão. Ao deixar de realizar esta prática, muitos sentimentos e coisas que poderiam ser resolvidas rapidamente acabam se acumulando e muitas vezes gerando uma verdadeira "bola de neve". Um exemplo real disto foi quando aqui no Elo7 tivemos um time que ficou algum tempo sem fazer retrospectivas e então em dado momento um dos membros propôs que começassem a fazer. Resultado: tiveram uma retrospectiva estendida por vários dias, de tantos assuntos importantes que se acumularam ao longo do tempo. Ou seja, algo que poderiam ter discutido e resolvido muito antes rapidamente, se tornou algo muito mais cansativo e demorado.

Reviewer: ```suggestion
Uma das principais razões para se fazer retrospectivas é a nova perspectiva que os membros ganham ao fazer autoavaliações, podendo discutir e ouvir vários pontos de vistas diferentes sobre si mesmo e sobre o projeto em questão. Ao deixar de realizar esta prática, muitos sentimentos e problemas que poderiam ser resolvidos rapidamente acabam se acumulando, muitas vezes gerando uma verdadeira "bola de neve". Um exemplo real que ocorreu no Elo7: um time ficou algum tempo sem fazer retrospectivas, até que em dado momento um dos membros propôs que começassem a fazer. Resultado: tiveram uma retrospectiva estendida por vários dias, devido a quantidade de assuntos importantes que se acumularam ao longo do tempo. Ou seja, algo que poderia ter sido discutido e resolvido muito antes, se tornou cansativo e demorado.
```

**Sugestão**

Além disso, são muitas as vantagens de se realizar retrospectivas em seu time, dentre elas podemos destacar:
* aumento na qualidade das entregas;
* aumento na produtividade do time;
* maior integração e engajamento do time;
* identificação e redução de falhas;
* visão mais ampla dos processos e objetivos;

> * tomadas de decisões mais eficientes;

Reviewer: ```suggestion
* tomadas de decisões mais eficientes.
```

## Como funciona?

> Uma retrospectiva acontece **sempre ao final de cada sprint** e, geralmente, dura em torno de 1 hora. Por exemplo, aqui no time do Nymeros no Elo7, cada sprint tem duração de duas semanas e finalizando nas terças-feiras, então deixamos agendadas as retrospectivas para as quartas-feiras às 16h. Além disso, existem alguns papéis e regras fundamentais para o sucesso de uma retrospectiva.

Reviewer: ```suggestion
Uma retrospectiva acontece **sempre ao final de cada sprint** e, geralmente, dura em torno de 1 hora. Por exemplo, aqui no time Nymeros no Elo7, cada sprint tem duração de duas semanas e se encerra às terças-feiras, sendo que as retrospectivas ficam agendadas para as quartas-feiras, às 16h. Além disso, existem alguns papéis e regras fundamentais para o sucesso de uma retrospectiva.
```
**Sugestão**
Outro ponto: quais são as regras fundamentais para o sucesso de uma retrospectiva?

Author: Adicionei esse ponto num tópico separado, quando puder me diz o que achou ou se seria melhor remover essa parte. Tenho minhas dúvidas se ficou bom 👀

### Participantes:

> * **Facilitador**: é a pessoa que irá moderar a reunião, ou seja, quem irá controlar o fluxo da discussão, definir o modelo da retrospectiva e direcionar os assuntos mais relevantes e monitorar o tempo. Nós do time Nymeros no Elo7, por exemplo, temos o costume de intercalar o facilitador entre os membros a cada sprint.

Reviewer: ```suggestion
* **Facilitador**: é a pessoa que irá moderar a reunião, ou seja, quem irá controlar o fluxo da discussão, definir o modelo da retrospectiva, direcionar os assuntos mais relevantes e monitorar o tempo. Nós do time Nymeros no Elo7, por exemplo, temos o costume de intercalar o facilitador a cada sprint.
```
**Sugestão**

> * **Membros**: são todos outros membros do time que irão participar da retrospectiva. É essencial que todos membros do time estejam presentes para que todos pontos de vista possam ser abordados. Outras pessoas também podem participar caso seja relevante, mas é importante ressaltar que isto é válido somente para quem realmente *precisa* estar lá, para que não haja desconfortos e inseguranças e impossibilite de haver uma conversa sincera e aberta sobre assuntos mais delicados.

Reviewer: ```suggestion
* **Membros**: são todos os outros membros do time que irão participar da retrospectiva. É essencial que todos membros estejam presentes para que todos pontos de vista sejam abordados. Outras pessoas também podem participar caso seja relevante, mas é importante ressaltar que isto é válido somente para quem realmente *precisa* estar lá, de forma que não haja desconfortos, inseguranças ou impossibilite de haver uma conversa sincera e aberta sobre assuntos mais delicados.
```
**Sugestão**

### Técnicas de Retrospectivas

> Existem várias técnicas de retrospectivas, e o facilitador definirá qual a melhor para aquele momento. O ideal é evitar fazer sempre a mesma coisa em todas retrospectivas, pois os desafios do time mudam com o tempo e a abordagem dessa dinâmica deve ser adaptada. Por exemplo, em sprints iniciais com um novo time, é interessante facilitar a construção desta equipe com técnicas de Team Building. Por outro lado, há ocasiões em que é necessário ajudar o time a dar e receber feedbacks. Em outros momentos, é necessário também olhar para frente com futurospectivas. Por isto é muito importante que o facilitador tenha uma boa visão de qual o contexto e o objetivo final em cada ocasião. Segue abaixo alguns exemplos de técnicas:

Reviewer: ```suggestion
Existem diversas técnicas de retrospectivas, e o facilitador é quem define qual a melhor para aquele momento. O ideal é evitar repetir a mesma técnica em todas as retrospectivas, pois os desafios do time mudam com o tempo e a abordagem dessa dinâmica deve ser adaptada. Por exemplo, em sprints iniciais com um novo time, é interessante facilitar a construção desta equipe com técnicas de _Team Building_. Por outro lado, há ocasiões em que é necessário ajudar o time a dar e receber _feedbacks_. Em outros momentos, é necessário também olhar para frente com futurospectivas. Por isto é muito importante que o facilitador tenha uma boa visão de qual o contexto e o objetivo final em cada ocasião. Segue abaixo alguns exemplos de técnicas:
```
**Sugestão**

Author: Opa boa observação colocar termos em inglês em itálico!

> * **Start, stop, continue**: é um modelo que foca nas coisas que estão funcionando (continue), nas que não estão funcionando (stop) e nas que o time deveria começar a fazer (start). Assim, ele se concentra no passado, presente e futuro ao mesmo tempo. Pode parecer uma técnica muito simples - mas funciona, pois ajuda os participantes a transmitir mais claramente seus problemas e preocupações, e, consequentemente, dando a todos uma visão geral do que aconteceu e não aconteceu, o que deve acontecer e o que precisa permanecer o mesmo.

Reviewer: ```suggestion
* **Start, stop, continue**: é um modelo que foca no que estão funcionando (_continue_), no que não está funcionando (_stop_) e no que o time deve começar a fazer (_start_). Assim, ele se concentra no passado, presente e futuro ao mesmo tempo. Pode parecer uma técnica muito simples, mas ajuda os participantes a transmitir mais claramente seus problemas e preocupações, dando a todos uma visão geral do que aconteceu, não aconteceu, o que deve acontecer e o que precisa permanecer o mesmo.
```
**Sugestão**

> * **4 L's (Liked, learned, lacked, longed for)**: este modelo é uma tentativa de capturar os pensamentos naturais que os participantes possam ter, visto que naturalmente tendemos a pensar e compartilhar em termos de coisas que apreciamos, lições aprendidas, coisas que faltaram e aquilo que gostaríamos de ter, e é isso que o 4L's irá trazer, de forma focada e útil.

Reviewer: ```suggestion
* **4 L's (Liked, learned, lacked, longed for)**: este modelo é uma tentativa de capturar os pensamentos naturais que os participantes possam ter, visto que naturalmente tendemos a pensar e compartilhar em termos de coisas que apreciamos, lições aprendidas, coisas que faltaram e aquilo que gostaríamos de ter. É isso que o 4L's irá trazer, de forma focada e útil.
```
**Sugestão**

> * **Mad, sad, glad**: é uma técnica que promove o bem-estar emocional, de forma que incentiva os participantes a discutirem sua jornada emocional durante a ultima sprint, compartilhando o que os fez se sentirem bravos (mad), tristes (sad) ou contentes (glad), sendo assim uma oportunidade de refletir sobre problemas e uma ótima forma de melhorar a satisfação no trabalho e criar um ambiente de equipe positivo. No entanto, é muito importante que todos se foquem nos eventos sem atribuir quaisquer culpas aos membros.

Reviewer: ```suggestion
* **Mad, sad, glad**: é uma técnica que promove o bem-estar emocional, de forma que incentiva os participantes a discutirem sua jornada emocional durante a ultima sprint, compartilhando o que os fez se sentirem bravos (_mad_), tristes (_sad_) ou contentes (_glad_). É uma oportunidade de refletir sobre problemas e uma ótima forma de melhorar a satisfação no trabalho, criando um ambiente positivo para a equipe. No entanto, é muito importante que todos se foquem nos eventos sem atribuir quaisquer culpas aos membros.
```
**Sugestão**

* **Sailboat**: é uma técnica de visualização cuja idéia é colocar o time para refletir sobre seus objetivos (ilha), impedimentos (âncoras), riscos (rochas) e em coisas que os favorecem (vento). Pode ser bastante interessante para se conduzir uma retrospectiva com mais de um time ao mesmo tempo, pois o nome ambas equipes serão colocadas no Sailboat, relembrando que estão todos no "mesmo barco" e navegando na mesma direção.

* **Hot Air Balloon**: é uma técnica interessante pois envolve também futurospectiva. Assim, na dinâmica, a equipe é incentivada a refletir sobre o passado e o futuro, discutindo os pontos positivos que levam o projeto para frente (ar quente), os pontos negativos que os atrasam (sacos de areia), os perigos adiante (tempestade) e nas coisas que esperam para o futuro (sol).

* **Starfish**: o objetivo desta atividade é ajudar as equipes a entender melhor o que fizeram de errado e como podem melhorar, permitindo que sejam avaliadas quais atividades precisam de mais ou de menos recursos e energia.

> * **KALM (Keep, Add, More, Less)**: esta atividade foca em práticas e processos em seu time, com o intuito de ajudar a equipe a refletir sobre as atividades atuais e seu valor, além de outras melhorias.

Reviewer: ```suggestion
* **KALM (Keep, Add, More, Less)**: esta atividade foca em práticas e processos do seu time, com o intuito de ajudar a equipe a refletir sobre as atividades atuais e seu valor, além de outras melhorias.
```
**Sugestão**

> São quiser conhecer ainda mais estas e outras técnicas existentes, recomendo a leitura destes sites: [Fun Retrospectives](https://www.funretrospectives.com/), [RetroAgil](https://retroagil.wordpress.com/) e [MetroRetro](https://metroretro.io/templates).

Reviewer: ```suggestion
Se quiser conhecer ainda mais sobre estas e outras técnicas existentes, recomendo a leitura destes sites: [Fun Retrospectives](https://www.funretrospectives.com/), [RetroAgil](https://retroagil.wordpress.com/) e [MetroRetro](https://metroretro.io/templates).
```
**Typo + sugestão**

### Ferramentas online para Retrospectivas

> Existem muitas ferramentas online que podem te auxiliar a realizar suas retrospectivas com seu time, aqui no Elo7 costumamos utilizar muito a  [MetroRetro](https://metroretro.io/), por conta de várias funcionalidades legais facilidades que este possui, além de ter um visual bem elegante. Mas além desta existem também muitas outras, algumas delas são:

Reviewer: ```suggestion
Existem muitas ferramentas online que podem te auxiliar a realizar suas retrospectivas com seu time. Aqui no Elo7 costumamos utilizar a [MetroRetro](https://metroretro.io/), por conta de várias funcionalidades legais que possui, além de ter um visual bem elegante. Além desta, existem também muitas outras:
```

**Sugestão**

---
title: Retrospectivas e sua importância
date: 2020-11-30
category: front-end
layout: post

> description: Neste post discutiremos os benefícios da adoção de retrospectivas no seu time.

Reviewer 2: Acho que poderia ser um pouco mais elaborada, mencionando também a importância de fazer com frequência e de usar diferentes técnicas para diferentes momentos. Ou seja: poderia ser um apanhado geral dos pontos que serão abordados no post.

Author: Feito!

Author: Feito!

authors: [mwingter]
tags:
  - retrospectivas
  - agile

>   - metodologias

Reviewer 2: Sugestão de mais uma tag: processos

> ---

Reviewer 2: Seria legal incluir uma imagem de capa para o post, talvez com uma foto de pessoas fazendo retrospectiva por exemplo.

Author: Feito!

> Retrospectivas são uma parte essencial das metodologias ágeis, que tem como um de seus pilares a melhoria contínua na produtividade do time. O objetivo deste post é mostrar a importância desta prática e como ela funciona, dando alguns exemplos de técnicas e de ferramentas online que nós do Elo7 temos utilizado para realizar a prática remotamente.

Reviewer 2: Além das pequenas correções de português, acho que a melhoria contínua não é apenas com foco em melhorar a produtividade do time, mas melhorar como time de uma forma geral, e melhorar outros aspectos fora do time também às vezes.

```suggestion
Retrospectivas são uma parte essencial das metodologias ágeis, que têm como um de seus pilares a melhoria contínua. O objetivo deste post é mostrar a importância desta prática e como ela funciona, dando alguns exemplos de técnicas e de ferramentas online que nós, do Elo7, temos utilizado para realizar a prática remotamente.
```

# Retrospectivas

> O objetivo principal de uma retrospectiva é avaliar todo o processo de trabalho ao final de cada sprint. Em outras palavras, isto significa um verdadeiro "lavar roupa suja": discutir o que deu ou não deu certo, o que poderia ser melhorado, o que continuar fazendo ou deixar de fazer. Com isto, é possível entender os pontos fortes e fracos, implementar estratégias e ações para superá-los, e assim garantir que a melhoria contínua aconteça, gerando um aumento na produtividade, qualidade e redução de falhas.

Reviewer 2: ```suggestion
O objetivo principal de uma retrospectiva é avaliar todo o processo de trabalho ao final de cada sprint. Em outras palavras, isto significa um verdadeiro "lavar roupa suja": discutir o que deu ou não deu certo, o que poderia ser melhorado, o que continuar fazendo ou deixar de fazer. Com isto, é possível entender os pontos fortes e fracos, implementar estratégias e ações para superá-los e, assim, garantir que a melhoria contínua aconteça, gerando um aumento na produtividade e na qualidade, além de uma redução de falhas.
```

> ## Porque fazer retrospectivas?

Reviewer 2: ```suggestion
## Por que fazer retrospectivas?
```

> Uma das principais razões para se fazer retrospectivas é a nova perspectiva que os membros ganham ao fazer autoavaliações, podendo discutir e ouvir vários pontos de vistas diferentes sobre si mesmo e sobre o projeto em questão. Ao deixar de realizar esta prática, muitos sentimentos e problemas que poderiam ser resolvidos rapidamente acabam se acumulando, muitas vezes gerando uma verdadeira "bola de neve". Um exemplo real que ocorreu no Elo7: um time ficou algum tempo sem fazer retrospectivas, até que em dado momento um dos membros propôs que começassem a fazer. Resultado: tiveram uma retrospectiva estendida por vários dias, devido à quantidade de assuntos importantes que se acumularam ao longo do tempo. Ou seja, algo que poderia ter sido discutido e resolvido muito antes, se tornou uma tarefa muito mais cansativa e demorada.

Reviewer 2: ```suggestion
Uma das principais razões para se fazer retrospectivas é a nova perspectiva que os membros ganham ao fazer autoavaliações, podendo discutir e ouvir vários pontos de vistas diferentes sobre si mesmos e sobre o projeto em questão. Ao deixar de realizar esta prática, muitos sentimentos e problemas que poderiam ser resolvidos rapidamente acabam se acumulando, muitas vezes gerando uma verdadeira "bola de neve". Um exemplo real que ocorreu no Elo7: um time ficou algum tempo sem fazer retrospectivas, até que em dado momento um dos membros propôs que as retrospectivas fossem retomadas. Resultado: tiveram uma retrospectiva estendida por vários dias, devido à quantidade de assuntos importantes que se acumularam ao longo do tempo. Ou seja, algo que poderia ter sido discutido e resolvido muito antes se tornou uma tarefa muito mais cansativa e demorada.
```

> Além disso, são muitas as vantagens de se realizar retrospectivas em seu time, dentre elas podemos destacar:

Reviewer 2: Acrescentar enter depois para o Hugo entender que é uma lista.
```suggestion
Além disso, são muitas as vantagens de se realizar retrospectivas em seu time. Dentre elas podemos destacar:

```

* aumento na qualidade das entregas;
* aumento na produtividade do time;
* maior integração e engajamento do time;
* identificação e redução de falhas;
* visão mais ampla dos processos e objetivos;
* tomadas de decisões mais eficientes.

## Como funciona?
Uma retrospectiva acontece **sempre ao final de cada sprint** e, geralmente, dura em torno de 1 hora. Por exemplo, aqui no time Nymeros no Elo7, cada sprint tem duração de duas semanas e se encerra às terças-feiras, sendo que as retrospectivas ficam agendadas para as quartas-feiras, às 16h. Além disso, existem alguns papéis e regras fundamentais para o sucesso de uma retrospectiva.

### Papéis (participantes):

> * **Facilitador**: é a pessoa que irá moderar a reunião, ou seja, quem irá controlar o fluxo da discussão, definir o modelo da retrospectiva, direcionar os assuntos mais relevantes e monitorar o tempo. Nós do time Nymeros no Elo7, por exemplo, temos o costume de intercalar o facilitador a cada sprint.

Reviewer 2: ```suggestion
* **Facilitador**: é a pessoa que irá moderar a reunião, ou seja, quem irá controlar o fluxo da discussão, definir o modelo da retrospectiva, direcionar os assuntos mais relevantes e monitorar o tempo. Nós do time Nymeros no Elo7, por exemplo, temos o costume de revezar o facilitador a cada sprint.
```

> * **Membros**: são todos os outros membros do time que irão participar da retrospectiva. É essencial que todos membros estejam presentes para que todos pontos de vista sejam abordados. Outras pessoas também podem participar caso seja relevante, mas é importante ressaltar que isto é válido somente para quem realmente *precisa* estar lá, de forma que não haja desconfortos, inseguranças ou impossibilite de haver uma conversa sincera e aberta sobre assuntos mais delicados.

Reviewer 2: ```suggestion
* **Membros**: são todos os outros membros do time que irão participar da retrospectiva. É essencial que todos os membros estejam presentes para que todos os pontos de vista sejam abordados. Outras pessoas também podem participar caso seja relevante, mas é importante ressaltar que isto é válido somente para quem realmente *precisa* estar lá, de forma que não haja desconfortos, inseguranças ou a impossibilidade de haver uma conversa sincera e aberta sobre assuntos mais delicados.
```

### Regras básicas

> Há duas regras básicas que devem ser sempre seguidas para o sucesso de qualquer tipo de retrospectiva:

Reviewer 2: Adiciona linha em branco a seguir para que a lista seja identificada corretamente.
```suggestion
Há duas regras básicas que devem ser sempre seguidas para o sucesso de qualquer tipo de retrospectiva:

```

> 1 - **Reconheça que todos deram o melhor de si**. É importante que todos tenham um senso de abertura e compaixão, para que se tenha uma conversa sem culpas nem julgamentos;

Reviewer 2: Melhor fazer usando a sintaxe do Markdown para listas numeradas, assim o HTML final fica mais semântico também.
```suggestion
1. **Reconheça que todos deram o melhor de si**. É importante que todos tenham um senso de abertura e compaixão, para que se tenha uma conversa sem culpas nem julgamentos;
```

Author: Ahh boa! 🚀

> 2 - **Saiba qual o objetivo**. Antes de começar a planejar a dinâmica, é essencial que se saiba quais os desafios e objetivos do time, para que se escolha a melhor técnica e tire o maior proveito da discussão.

Reviewer 2: Idem (+ partícula "se" faltando)
```suggestion
2. **Saiba qual o objetivo**. Antes de começar a planejar a dinâmica, é essencial que se saiba quais os desafios e objetivos do time, para que se escolha a melhor técnica e se tire o maior proveito da discussão.
```

### Técnicas de Retrospectivas

> Existem diversas técnicas de retrospectivas, e o facilitador é quem define qual a melhor para aquele momento. O ideal é evitar repetir a mesma técnica em todas as retrospectivas, pois os desafios do time mudam com o tempo e a abordagem desta dinâmica deve ser adaptada. Por exemplo, em sprints iniciais com um novo time, é interessante facilitar a construção desta equipe com técnicas de _Team Building_. Por outro lado, há ocasiões em que é necessário ajudar o time a dar e receber _feedbacks_. Em outros momentos, é necessário também olhar para frente com futurospectivas. Por isto é muito importante que o facilitador tenha uma boa visão de qual o contexto e o objetivo final em cada ocasião. Segue abaixo alguns exemplos de técnicas:

Reviewer 2: Talvez fosse interessante mencionar também que a variação da técnica pode motivar os membros do time a participar, já que o ritual deixa de ser repetitivo.

> Existem diversas técnicas de retrospectivas, e o facilitador é quem define qual a melhor para aquele momento. O ideal é evitar repetir a mesma técnica em todas as retrospectivas, pois os desafios do time mudam com o tempo e a abordagem desta dinâmica deve ser adaptada. Por exemplo, em sprints iniciais com um novo time, é interessante facilitar a construção desta equipe com técnicas de _Team Building_. Por outro lado, há ocasiões em que é necessário ajudar o time a dar e receber _feedbacks_. Em outros momentos, é necessário também olhar para frente com futurospectivas. Por isto é muito importante que o facilitador tenha uma boa visão de qual o contexto e o objetivo final em cada ocasião. Segue abaixo alguns exemplos de técnicas:

Reviewer 2: ```suggestion
Existem diversas técnicas de retrospectivas, e o facilitador é quem define qual a melhor para aquele momento. O ideal é evitar repetir a mesma técnica em todas as retrospectivas, pois os desafios do time mudam com o tempo e a abordagem desta dinâmica deve ser adaptada. Por exemplo, em sprints iniciais com um novo time, é interessante facilitar a construção desta equipe com técnicas de _Team Building_. Por outro lado, há ocasiões em que é necessário ajudar o time a dar e receber _feedbacks_. Em outros momentos, é necessário também olhar para frente com futurospectivas. Por isto é muito importante que o facilitador tenha uma boa visão de qual o contexto e o objetivo final em cada ocasião. Seguem abaixo alguns exemplos de técnicas:
```

> * **Start, stop, continue**: é um modelo que foca no que está funcionando (_continue_), no que não está funcionando (_stop_) e no que o time deveria começar a fazer (_start_). Assim, ele se concentra no passado, presente e futuro ao mesmo tempo. Pode parecer uma técnica muito simples, mas ajuda os participantes a transmitir mais claramente seus problemas e preocupações, dando a todos uma visão geral do que aconteceu ou não aconteceu, o que deve acontecer e o que precisa permanecer o mesmo.

Reviewer 2: ```suggestion
* **Start, stop, continue**: é um modelo que foca no que está funcionando (_continue_), no que não está funcionando (_stop_) e no que o time deveria começar a fazer (_start_). Assim, ele se concentra no passado, presente e futuro ao mesmo tempo. Pode parecer uma técnica muito simples, mas ajuda os participantes a transmitirem mais claramente seus problemas e preocupações, dando a todos uma visão geral do que aconteceu ou não aconteceu, o que deve acontecer e o que precisa permanecer como está.
```

* **4 L's (Liked, learned, lacked, longed for)**: este modelo é uma tentativa de capturar os pensamentos naturais que os participantes possam ter, visto que naturalmente tendemos a pensar e compartilhar em termos de coisas que apreciamos, lições aprendidas, coisas que faltaram e aquilo que gostaríamos de ter. É isso que o 4L's irá trazer, de forma focada e útil.

* **Mad, sad, glad**: é uma técnica que promove o bem-estar emocional, de forma que incentiva os participantes a discutirem sua jornada emocional durante a ultima sprint, compartilhando o que os fez se sentirem bravos (_mad_), tristes (_sad_) ou contentes (_glad_). É uma oportunidade de refletir sobre problemas e uma ótima forma de melhorar a satisfação no trabalho, criando um ambiente positivo para a equipe. No entanto, é muito importante que todos se foquem nos eventos sem atribuir quaisquer culpas aos membros.

> * **Sailboat**: é uma técnica de visualização cuja idéia é colocar o time para refletir sobre seus objetivos (ilha), impedimentos (âncoras), riscos (rochas) e em coisas que os favorecem (vento). Pode ser bastante interessante para se conduzir uma retrospectiva com mais de um time ao mesmo tempo, pois o nome ambas equipes serão colocadas no Sailboat, relembrando que estão todos no "mesmo barco" e navegando na mesma direção.

Reviewer 2: ```suggestion
* **Sailboat**: é uma técnica de visualização cuja ideia é colocar o time para refletir sobre seus objetivos (ilha), impedimentos (âncoras), riscos (rochas) e coisas que os favorecem (vento). Pode ser bastante interessante para se conduzir uma retrospectiva com mais de um time ao mesmo tempo, pois os nomes de ambas as equipes serão colocados no Sailboat, relembrando que estão todos no "mesmo barco" e navegando na mesma direção.
```

> * **Sailboat**: é uma técnica de visualização cuja idéia é colocar o time para refletir sobre seus objetivos (ilha), impedimentos (âncoras), riscos (rochas) e em coisas que os favorecem (vento). Pode ser bastante interessante para se conduzir uma retrospectiva com mais de um time ao mesmo tempo, pois o nome ambas equipes serão colocadas no Sailboat, relembrando que estão todos no "mesmo barco" e navegando na mesma direção.

Reviewer 2: Por ser uma técnica de visualização, talvez fosse interessante incluir uma imagem para ilustrá-la.

> * **Hot Air Balloon**: é uma técnica interessante pois envolve também futurospectiva. Assim, na dinâmica, a equipe é incentivada a refletir sobre o passado e o futuro, discutindo os pontos positivos que levam o projeto para frente (ar quente), os pontos negativos que os atrasam (sacos de areia), os perigos adiante (tempestade) e nas coisas que esperam para o futuro (sol).

Reviewer 2: Também é uma técnica de visualização, então poderia ser interessante incluir uma imagem também.

> * **Starfish**: o objetivo desta atividade é ajudar as equipes a entender melhor o que fizeram de errado e como podem melhorar, permitindo que sejam avaliadas quais atividades precisam de mais ou de menos recursos e energia.

Reviewer 2: Acho que poderia detalhar um pouco mais, explicando como fica dividida a lousa.

* **KALM (Keep, Add, More, Less)**: esta atividade foca em práticas e processos do seu time, com o intuito de ajudar a equipe a refletir sobre as atividades atuais e seu valor, além de outras melhorias.

Se quiser conhecer ainda mais sobre estas e outras técnicas existentes, recomendo a leitura destes sites: [Fun Retrospectives](https://www.funretrospectives.com/), [RetroAgil](https://retroagil.wordpress.com/) e [MetroRetro](https://metroretro.io/templates).

### Ferramentas online para Retrospectivas

> Existem muitas ferramentas online que podem te auxiliar a realizar suas retrospectivas com seu time. Aqui no Elo7 costumamos utilizar a [MetroRetro](https://metroretro.io/), por conta de várias funcionalidades legais que possui, além de ter um visual super atrativo. Além desta existem também muitas outras:

Reviewer 2: Adicionar linha em branco para que a lista seja corretamente identificada.
```suggestion
Existem muitas ferramentas online que podem te auxiliar a realizar suas retrospectivas com seu time. Aqui no Elo7 costumamos utilizar a [MetroRetro](https://metroretro.io/), por conta de várias funcionalidades legais que possui, além de ter um visual super atrativo. Além desta existem também muitas outras:

```
